dev/wlp-jakartaee-transform/rules/jakarta-xml-master.properties

```diff
@@ -5,6 +5,7 @@ basic-constraints.xml=jakarta-xml-dd.properties
 ejb-jar.xml=jakarta-xml-dd.properties
 permissions.xml=jakarta-xml-dd.properties
 ra.xml=jakarta-xml-dd.properties
+server.xml=jakarta-xml-server.properties
```

Comment:
Will this transform files that end with server.xml?  For example there are test suites that user <method-name>-server.xml

Reply:
The new line enables the transformation of files named `server.xml`.